src/key/local.rs

```diff
@@ -40,8 +38,6 @@ pub struct KeyPairGeneratorLocalConfig {
 pub struct LocalCreator {
     /// The type of key to be created.
     key_type: KeyType,
-    /// The name associated with the key.
-    name: String,
     /// The file path where the private key will be stored.
     path: PathBuf,
```

Comment:
Nit: `file_path`

Reply:
Done in https://github.com/newrelic/newrelic-auth-rs/pull/52/commits/c461030bcf1ff33fb2fd0afb43eeebcbc096ff4f!